### PR TITLE
Remove jQuery dependency

### LINKED ISSUE #41355: Stock Price Checker: Content Security Policy requirement breaks the front end of the application.
**Describe your problem and how to reproduce it:**
The requirement "You should set the content security policies to only allow loading of scripts and CSS from your server." breaks the front end index.html page, if it is implemented properly. 

The helmet security middleware should be at the top of the `app.use()` chain in order to provide maximum benefit to the application. However, if you do this, then the front end index.html page is broken because it relies on jquery to be downloaded from `https://code.jquery.com/jquery-2.2.1.min.js`

This can be worked around and pass the requirement with a couple of methods, the first is to use the helmet middleware after the route for '/' and before any other routes are defined. The second method is to download a copy of jquery, place it in the project and change index.html to reference the copy.

Of the two methods, the second one is more secure as all of your routes have the content security policy and you are controlling the script that is being loaded. This does come at the cost of more disk space and if this was a real world application building a process to watch for updates and/or security fixes to jquery and getting those fixes pushed to the application.

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/information-security/information-security-projects/stock-price-checker

**Tell us about your browser and operating system:**
* Browser Name: Mozilla Firefox
* Browser Version: 86.0
* Operating System: Windows 10

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
This screenshot shows the network activity without a CSP. Both the favicon from freeCodeCamp and jquery are loaded. The get request is initiated by jquery.

no-csp-policy

This screen shot shows the activity when the CSP policy for scripts is set to self. Neither the favicon or jquery are loaded and submitting the form just causes the page to refresh without a call to /api/stock-prices.

self-csp-policy

### PR DESCRIPTION
**Pull request by santi224m**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes [#41355](https://github.com/freeCodeCamp/freeCodeCamp/issues/41355) on freeCodeCamp/freeCodeCamp repository.

This replaces jQuery with a JavaScript file so that when users set the content security policies to only allow files from the same server, it won't break the page.
It also moves page styling from the html file to a css file.

### CODE REVIEW

**Review by gikf — COMMENTED** [review]
I've tested this on fresh boilerplate copy and it seemed fine.

Could you change the indentations to use two spaces per level?

I'm a bit confused with code formatting of couple `<code>` tags, placing angle brackets on different line, but I'm gonna refer to other's opinion regarding that.

**Comment by santi224m** [comment]
I changed the indentations to two spaces on the three files and I also fixed the formatting on the ```<code>``` tags. Let me know if you want me to change anything else.

File: views/index.html [hunk 1]
```diff
@@ -2,76 +2,60 @@
...
+        <input type="checkbox" name="like" value="true" /> Like both?
+        <br />
         <button type="submit">Get Price!</button>
       </form>
-      <code id='jsonResult'></code>
+      <code id="jsonResult"></code>
     </div>
-    <hr style='margin: 50px; margin-top: 200px'>
+    <hr class="m-50 mt-200" />
     <!-- Your web-app is https, so your scripts need to be too -->
```
Comment (gikf) [inline views/index.html]:
```suggestion
```
After removing externally linked jquery, this is not really needed for anything.

**Review by gikf — CHANGES_REQUESTED** [review]
Thanks for the update @santi224m.

Just one more, small suggestion and I think I'd be okay with the rest.

**Comment by santi224m** [comment]
@gikf I removed the comment, I think it should all be good now

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Pull request merged**